stock/src/main/java/nl/tudelft/wdm/group1/stock/StockItem.java

```diff
@@ -1,22 +1,48 @@
 package nl.tudelft.wdm.group1.stock;
 
+import jdk.nashorn.internal.objects.annotations.Constructor;
+
 import java.util.UUID;
 
 public class StockItem {
     private UUID id;
+    private int stock;
 
     public StockItem() {
-        id = UUID.randomUUID(); // TODO: Remove this line as soon as the non-default constructor is created
+
+    }
+
+    public StockItem(final int stock) {
+        this.id = UUID.randomUUID();
+        this.stock = stock;
     }
 
     public UUID getId() {
         return id;
     }
 
+    public int getStock() { return this.stock; }
+
+    public void addStock(final int amount) {
+        if (amount > 0) this.stock += amount; // TODO: maybe create a new exception for this?
```

Comment:
Why do we need this if statement here?

Reply:
I want to make sure the amount is positive

Reply:
Ahh, I misread this as `if (stock > 0)`

In that case I would throw an exception in the else case. See https://github.com/casperboone/wdm-kafka-microservices/blob/master/users/src/main/java/nl/tudelft/wdm/group1/users/User.java for an example (credit subtraction/addition is very similar)

Reply:
alright then